vault/diagnose/helpers_test.go

```diff
@@ -70,6 +75,7 @@ func makeCoffee(ctx context.Context) error {
 
 	SpotCheck(ctx, "pick-scone", pickScone)
 
+	Test(ctx, "dispose-grounds", Skippable("dispose-grounds", disposeGrounds))
```

Comment:
just to verify, Test does the appropriate thing if given a function that returns `nil`?

Reply:
Yeah, marks the span successful (but will show up as Skipped if the skipped event was added)